src/pages/burn-out.md

```diff
@@ -7,12 +7,28 @@ color: "yellow"
 
 ## Intro
 
-Maintainers can get burned-out.
+Let's not kid ourselves. Open Source can be a miserable place to work in. 
```

Comment:
The spelling of `Open Source` seems a bit off.
Should it be like this? Or `Open-source`?

Reply:
Hard to say what's "correct", you find all kinds of spellings, even in our page.
I went with "Open-Source", but no hill I want to die on 😆

Reply:
Yeah, I struggle with this one a lot myself as well.